src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs

```diff
@@ -238,7 +227,7 @@ private Checksum[] AddGlobalAssets(CancellationToken cancellationToken)
 
                 using (Logger.LogBlock(FunctionId.RemoteHostClientService_AddGlobalAssetsAsync, cancellationToken))
                 {
-                    var snapshotService = _workspace.Services.GetRequiredService<IRemotableDataService>();
+                    var snapshotService = _workspace.Services.GetService<IRemotableDataService>();
```

Comment:
why no longer required?

Reply:
it doesn't look like the code is safe for this actually reutrning null (it will crash a few lines below).  so why not keep this as GetRequired?

Reply:
Reverted the commit.